Libs/MRML/Core/vtkMRMLTextNode.h

```diff
@@ -0,0 +1,104 @@
+/*=auto=========================================================================
+
+  Portions (c) Copyright 2009 Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See Doc/copyright/copyright.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Program:   3D Slicer
+  Module:    $RCSfile: vtkMRMLTextNode.h,v $
+  Date:      $Date: 2006/03/19 17:12:29 $
+  Version:   $Revision: 1.3 $
+
+=========================================================================auto=*/
+
+#ifndef __vtkMRMLTextNode_h
+#define __vtkMRMLTextNode_h
+
+// MRML includes
+#include <vtkMRMLStorableNode.h>
+
+// STD includes
+#include <sstream>
+
+class  VTK_MRML_EXPORT vtkMRMLTextNode : public vtkMRMLStorableNode
+{
+public:
+  enum
+  {
+    ENCODING_US_ASCII = 3,
+    ENCODING_ISO_8859_1 = 4,
+    ENCODING_LATIN1 = ENCODING_ISO_8859_1 // alias
+                      // see other codes at http://www.iana.org/assignments/character-sets/character-sets.xhtml
```

Comment:
It would be nice to also support UTF8 and add test for it.

Reply:
Added a test for UTF-8, and decided to just use the VTK_ENCODING_X enum to represent encoding.